### ISSUE TITLE
about the theory

### ISSUE THREAD

**Issue by z1zzzz1** [wgcban/ddpm-cd#22]
Thank you very much for your work, but I still have some problems with the paper: as far as I know the diffusion model is a generative model that can get images similar to the training dataset from random noise, but I don't understand this What is the relationship between this function and your thesis, that is to say, I don't understand the meaning of you using a large number of pictures to pre-train a diffusion model, and then use it to extract the features of the image.
(I don't mean any offense)
Looking forward to your reply, thank you

**Comment by ZHUYMGeo** [comment]
I have the same question, what is the motivation for the research

**Comment by wgcban** [comment]
Hi @ZHUYMGeo @z1zzzz1 

Please check the last two paragraph of the INTRO for the motivation.
```
Remote sensing pre-training [49] is one of the most widely adopted techniques for this purpose. With the emergence of large-scale areal-scene recognition datasets such as MillionAID [30], fMoW [13] and BigEarthNet [47], one can pre-train the encoder of the CD model to capture the key feature representations (i.e., semantics) of remote sensing images. However, the need for image-level class labels hinders its ability to train on on millions of unlabeled and uncurated remote sensing images, which has motivated the researchers to explore self-supervised pre-training methods [11, 12, 54] such as SeCo [33], where the presence of seasonal changes in aerial scenes is used to ensure consistency between positive samples.

Different from the above pre-training techniques, in this paper, we show that the diffusion probabilistic models [44, 22, 35] can also be used to extract key semantics of remote sensing images. Diffusion models are directed graphical models parameterized by a Markov chain (see figure 2). We train a diffusion model using a variational inference procedure on a large collection of off-the-shelf remote sensing images to generate images that are closer to real images over the finite time. Figure 1 shows some sample images generated from the diffusion model. We can see that the generated images are very closer to the actual remote sensing images and contains key objects find in remote sensing images, such as buildings, trees, vegetation, roads, water surfaces, etc. Therefore, diffusion model is powerful enough to extract key features of a given image. While any generative model (such as VAEs [25, 26] or GANs [20, 46]) can be trained to learn key semantics in a similar way, based on our experiments, diffusion models have been shown to produce better feature representations. From the experiments, we observe that the representational power oF VAE and GANs are limited compared to diffusion models, resulting in lower CD performance (see the supplementary material for more details). Since diffusion models can be used to extract better semantics of given a image, we fine-tune a light-weight CD classifier with the limited pixel-level change labels by utilizing multi-scale semantics (i.e., deep feature representations) from the diffusion model as its inputs. Experiments conducted on multiple CD datasets show that the proposed method achieves significant performance improvement over the state-of-the-art (SOTA) CD methods, demonstrating that diffusion probabilistic models can be used not only to sample images closer to the training data distribution, but also to extract deep semantics that are useful for low-level vision tasks.

**Comment by wgcban** [comment]
If you have any specific questions about the motivation, please let us know.
